docs/cvl/sorts.md

```diff
@@ -1,44 +1,62 @@
-Uninterpreted Sorts
-===================
+# Uninterpreted Sorts
 
-The syntax for `sort` declarations is given by the following [EBNF grammar](ebnf-syntax):
+CVL specifications support both Solidity primitives (`uint256`, `address`, etc.) and custom types (e.g., `mathint`). Solidity types are _interpreted_, meaning they have specific semantics, such as arithmetic or comparison operations. However, in some cases, it is beneficial to use _uninterpreted sorts_, which do not carry the semantics associated with interpretation.
 
-```
-sort ::= "sort" id ";"
+## Syntax for Uninterpreted Sorts
+
+To declare an uninterpreted sort in CVL, use the following syntax:
+
+```cvl
+Sort MyUninterpSort;
```

Comment:
Should be `sort` and not `Sort` (testing on certora-cli-beta 6.3.0). Also see other instances below.